components/map/InterventionMap.tsx

```diff
@@ -71,49 +90,119 @@ function InterventionMap({
-        </Marker>
-      ))}
-      {fireFighterMeans.map((mean) => (
-        <Marker
-          key={mean.id}
-          coordinate={{
-            latitude: mean.means.location.latitude,
-            longitude: mean.means.location.longitude,
-          }}
-        >
-          <FireFighterVehicle
-            color={getDangerCodeColor(mean.danger_code)}
-            name={mean.means.label}
-            dashed={!mean.is_on_site}
+    <Box flex={1}>
+      <MapBackground handlePress={handlePress} initialRegion={interventionLocation}>
+        {data?.records.map((item: any) => (
+          <Marker
+            key={item.recordid}
+            coordinate={{
+              latitude: item.fields.geo_point_2d[0],
+              longitude: item.fields.geo_point_2d[1],
+            }}
+            title={item.fields.type}
+          >
+            <CustomCircle size={{ height: 30, width: 30 }} color="#2D3ED3" fill />
+          </Marker>
+        ))}
+        {fireFighterMeans.map((mean) => (
+          <Marker
+            key={mean.id}
+            coordinate={{
+              latitude: mean.means.location.latitude,
+              longitude: mean.means.location.longitude,
+            }}
+            title="Modifier moyen"
+            onCalloutPress={() => handleFireFighterVehiclePress(mean)}
+          >
+            <FireFighterVehicle
+              color={getDangerCodeColor(mean.danger_code)}
+              name={mean.means.label}
+              dashed={!mean.is_on_site}
+            />
+          </Marker>
+        ))}
+        {otherMeans.map((mean) => (
+          <Marker
+            key={mean.id}
+            coordinate={{
+              latitude: mean.location.latitude,
+              longitude: mean.location.longitude,
+            }}
+            title="X"
+            onCalloutPress={() => deleteOtherMeanMutation(mean.id)}
+          >
+            {selectRightSymbol(mean.category, mean.danger_code)}
+          </Marker>
+        ))}
+        <Polyline coordinates={newPolyline} strokeColor={drawingsColor} strokeWidth={3} />
+        {currentPolylines.map((mean) => (
+          <Polyline
+            key={mean.id}
+            coordinates={mean.points}
+            strokeColor={getDangerCodeColor(mean.danger_code)}
+            strokeWidth={3}
+            onPress={() => setPressedPolylineId(mean.id)}
+            tappable
           />
-        </Marker>
-      ))}
-      {otherMeans.map((mean) => (
-        <Marker
-          key={mean.id}
-          coordinate={{
-            latitude: mean.location.latitude,
-            longitude: mean.location.longitude,
-          }}
-        >
-          {selectRightSymbol(mean.category, mean.danger_code)}
-        </Marker>
-      ))}
-    </MapBackground>
+        ))}
+      </MapBackground>
+      <Fab
+        renderInPortal={false}
+        placement="bottom-right"
+        backgroundColor={polylineDrawMode ? 'white' : '#19837C'}
```

Comment:
C'est pas que relatif à cette PR mais au lieu de mettre les couleurs en hexa à chaque fois, il faudrait les mettre dans un objet
Par exemple:
```ts
const Colors = {
    blue: '#0000FF'
}
```

Reply:
On va dire à Inès de faire ça